cmd/build/v2/build.go

```diff
@@ -88,11 +91,12 @@ type OktetoBuilder struct {
 }
 
 // NewBuilder creates a new okteto builder
-func NewBuilder(builder OktetoBuilderInterface, registry oktetoRegistryInterface, analyticsTracker analyticsTrackerInterface) *OktetoBuilder {
+func NewBuilder(builder OktetoBuilderInterface, registry oktetoRegistryInterface, analyticsTracker analyticsTrackerInterface, okCtx *buildCtx.OktetoContext) *OktetoBuilder {
```

Comment:
Why do we need to import `*buildCtx.OktetoContext` I think we should be going with `buildCtx.OktetoContextInterface`